nbs/models.moment.ipynb

```diff
@@ -0,0 +1,1518 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #112.                "AutonLab/MOMENT-1-large", 
</pre><p>i think here it's <code>model</code> instead of  <code>AutonLab/MOMENT-1-large</code>.  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1011/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Good catch